Documentation/contributing/development/dev_setup.rst

```diff
@@ -36,7 +36,7 @@ contribute to Cilium:
 +--------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
 | `go <https://golang.org/dl/>`_                               | |GO_RELEASE|                 | N/A (OS-specific)                                               |
 +--------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
-+ `ginkgo <https://github.com/onsi/ginkgo>`__                  | >= 1.4.0                     | ``go install github.com/onsi/ginkgo/ginkgo@latest``             |
++ `ginkgo <https://github.com/onsi/ginkgo>`__                  | >= 1.4.0 and < 2.0.0         | ``go install github.com/onsi/ginkgo/ginkgo@latest``             |
```

Comment:
Should we also update the command?

Reply:
By default, go install github.com/onsi/gingko/ginkgo@latest will only install the latest v1 version, at the time of writing, this will be v1.16.5. So I just keep the command the same.

Reply:
But won't that change soon-ish?

Reply:
According to https://onsi.github.io/ginkgo/MIGRATING_TO_V2 v2 lives in its own module `github.com/onsi/ginkgo/v2` and v1 will no longer receive any updates. If they follow the Go module versioning guidelines (https://go.dev/blog/v2-go-modules), v1 and v2 should remain as two separate packages. So tl;dr it should be fine to keep the command as is.